config/locales/en.yml

```diff
@@ -1064,7 +1056,7 @@ en:
       new:
         add_member_to_project: Add member to project
         description: You are adding a new member to project %{name}
-        empty_state: Empty state
+        empty_state: There are no users with the provided email or id
```

Comment:
**nitpick (non-blocking):** The other places we have a capital `ID`

Reply:
Fixed in 8b72140